### PR TITLE
Make Mininuke real-er

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary

None

#### Purpose of change

The mininuke didn't have its real dimensions. (It's based off the w54 warhead.) The mininuke hack doesn't spawn from the Dispatch drones. 

#### Describe the solution

Obsoletes mininuke hacks, broken mininuke hacks, inactive mininuke hacks, fake items associated with these. Re-sizes the mininuke and redoes its description.

Adds a cute carrying strap to the mininuke. 
image

#### Describe alternatives you've considered

Deletion, but I do want these to spawn on super special ops guys opposing XEDRA.

#### Testing
image

image

image

Loaded game, spawned items, blew up the thing. Still works.

#### Additional context

<https://www.thedrive.com/the-war-zone/special-forces-parachuted-with-nukes-strapped-to-them-during-the-cold-war>

### CODE REVIEW

File: data/json/obsoletion/obsolete_monsters.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "diff": 40,
+    "hp": 20,
+    "speed": 150,
+    "color": "light_green_magenta",
+    "revert_to_itype": "bot_mininuke_hack",
+    "starting_ammo": { "mininuke_payload": 1 },
+    "special_attacks": [ [ "KAMIKAZE", 0 ] ],
+    "armor": { "bash": 1, "cut": 6, "bullet": 5 }
+  }
+]
```
Comment (Karol1223) [inline data/json/obsoletion/obsolete_monsters.json:17]: All of this is unnecessary. We don't do monster obsoletion nowadays. Removed monsters silently transform into breathers.

**Review by Karol1223 — CHANGES_REQUESTED** [review]

File: data/json/obsoletion/obsolete_monsters.json (hunk 1 above)
Comment (tenmillimaster) [reply data/json/obsoletion/obsolete_monsters.json:17]: What about the transform target for the inactive mininuke hack?

Comment (Karol1223) [reply data/json/obsoletion/obsolete_monsters.json:17]: You can test what happens ingame, but I am 99% sure that it'll just spawn a breather without an error. That's what happens to monsters stored inside pet carriers - they still show up with their old name, but when you release them they come out as breathers.

**Comment by pantherpusher** [comment]
well now I want to own one of them bags

File: data/json/obsoletion/obsolete_monsters.json (hunk 1 above)
Comment (tenmillimaster) [reply data/json/obsoletion/obsolete_monsters.json:17]: It doesn't work.

**Pull request merged**